ros2param/ros2param/verb/list.py

```diff
@@ -97,7 +102,14 @@ def main(self, *, args):  # noqa: D102
                         print(f'{node_name.full_name}:')
                     response = future.result()
                     for name in sorted(response.result.names):
-                        print(f'  {name}')
+                        if args.param_type is True:
+                            resp = call_describe_parameters(
```

Comment:
Or maybe we can call this before with parameter_names=None, then search the parameter name which matches with 'name' from list service response.

Reply:
Yeah, I think that would be better.  Doing a separate service call for every parameter is going to be expensive and possibly slow (particularly if the node is on a different machine on the network).

Reply:
Done